ui/imports/shared/stores/RootStore.qml

```diff
@@ -36,7 +36,8 @@ QtObject {
 
     property var history: walletSectionTransactions
     property var historyTransactions: walletSectionTransactions.model
-    property bool isNonArchivalNode:  history.isNonArchivalNode
+    property bool isNonArchivalNode: typeof history !== "undefined" ? history.isNonArchivalNode
```

Comment:
Not that I want to be spoiling the fun here... but wouldn't it be better to actually postpone these warning fix PRs for after Qt 5.15? 

These could then easily be rewritten as:
```qml
property bool isNonArchivalNode: history.isNonArchivalNode ?? false
```

wdyt @jrainville ?

Reply:
Nullish Coalescing will not solve the issue addressed in this PR.

`property bool isNonArchivalNode: history.isNonArchivalNode ?? false` will still result in a warning if `history` is not defined

Reply:
I stand corrected; `property bool isNonArchivalNode: history?.isNonArchivalNode || false`

Reply:
Yeah, optional chaining would help a lot. Is it accessible in Qt 5.15.x?

Reply:
In Qt6 :/ https://codereview.qt-project.org/c/qt/qtdeclarative/+/341496